### PR TITLE
Add table aws_lambda_function_metric_invocations_daily

### PR DESCRIPTION
**Pull request by sankeyraut**
# Example query results
<details>
  <summary>Results</summary>
  
```

> select
  name,
  timestamp,
  sum
from
  aws_lambda_function_metric_invocations_daily
order by
  name,
  timestamp;
+-----------------------------------------------------------------+---------------------+------+
| name                                                            | timestamp           | sum  |
+-----------------------------------------------------------------+---------------------+------+
| GitStack-GitLookerA37793FB-ZYgxqb22sQii                         | 2021-08-17 06:00:00 | 56   |
| GitStack-GitLookerA37793FB-ZYgxqb22sQii                         | 2021-08-18 06:00:00 | 2    |
| GitStack-GitLookerA37793FB-ZYgxqb22sQii                         | 2021-08-19 06:00:00 | 1    |
| GitStack-GitLookerA37793FB-ZYgxqb22sQii                         | 2021-08-20 06:00:00 | 1    |
| GitStack-GitLookerA37793FB-ZYgxqb22sQii                         | 2021-08-21 06:00:00 | 2    |
| GitStack-GitLookerA37793FB-ZYgxqb22sQii                         | 2021-08-22 06:00:00 | 1    |
| GitStack-GitLookerA37793FB-ZYgxqb22sQii                         | 2021-08-23 06:00:00 | 8    |
+-----------------------------------------------------------------+---------------------+------+



> select
  name,
  timestamp,
  round(sum::numeric,2) as sum_invocations,
  sample_count
from
  aws_lambda_function_metric_invocations_daily
where
  date_part('day', now() - timestamp) <=3
and sum > 10
order by
  name,
  timestamp;
+-----------------------------------------------------+---------------------+-----------------+--------------+
| name                                                | timestamp           | sum_invocations | sample_count |
+-----------------------------------------------------+---------------------+-----------------+--------------+
| SC-XUG3HH5R-AutoUpdateLambda-1D97WFPMN              | 2021-08-21 06:00:00 | 24.00           | 24           |
| SC-XUG3HH5R-AutoUpdateLambda-1D97WFPMN              | 2021-08-22 06:00:00 | 24.00           | 24           |
| SC-XUG3HH5R-AutoUpdateLambda-1D97WFPMN              | 2021-08-23 06:00:00 | 24.00           | 24           |
| SC-XUG3HH5R-SCLambda-M8W8IEP5M                      | 2021-08-21 06:00:00 | 74.00           | 74           |
| SC-XUG3HH5R-SCLambda-M8W8IEP5M                      | 2021-08-22 06:00:00 | 75.00           | 75           |
| SC-XUG3HH5R-SCLambda-M8W8IEP5M                      | 2021-08-23 06:00:00 | 75.00           | 75           |
+-----------------------------------------------------+---------------------+-----------------+--------------+
```
</details>

### CODE REVIEW

File: aws/table_aws_lambda_function_metric_invocations_daily.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	"github.com/aws/aws-sdk-go/service/lambda"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+func tableAwsLambdaFunctionMetricInvocationsDaily(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_lambda_function_metric_errors_daily",
```
Comment (bigdatasourav) [inline aws/table_aws_lambda_function_metric_invocations_daily.go:15]:
```suggestion
		Name:        "aws_lambda_function_metric_invocations_daily",
```

File: aws/table_aws_lambda_function_metric_invocations_daily.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+		Description: "AWS Lambda Function Cloudwatch Metrics - Errors (Daily)",
```
Comment (bigdatasourav) [inline aws/table_aws_lambda_function_metric_invocations_daily.go:16]:
```suggestion
		Description: "AWS Lambda Function Cloudwatch Metrics - Invocations (Daily)",
```

File: docs/tables/aws_lambda_function_metric_invocations_daily.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Table: aws_lambda_function_metric_invocations_daily
+
+Amazon CloudWatch Metrics provide data about the performance of your systems.  The `aws_lambda_function_metric_invocations_daily` table provides metric statistics at 24 hour intervals for the last year.
```
Comment (bigdatasourav) [inline docs/tables/aws_lambda_function_metric_invocations_daily.md:3]:
```suggestion
Amazon CloudWatch Metrics provide data about the performance of your systems. The `aws_lambda_function_metric_invocations_daily` table provides metric statistics at 24 hour intervals for the last year.
```

File: docs/tables/aws_lambda_function_metric_invocations_daily.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+
```
Comment (bigdatasourav) [inline docs/tables/aws_lambda_function_metric_invocations_daily.md:5]:
```suggestion
```

File: docs/tables/aws_lambda_function_metric_invocations_daily.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  sum
+from
+  aws_lambda_function_metric_invocations_daily
+order by
+  name,
+  timestamp;
+```
+
+
+
```
Comment (bigdatasourav) [inline docs/tables/aws_lambda_function_metric_invocations_daily.md:23]:
```suggestion
```

Comment (bigdatasourav) [inline docs/tables/aws_lambda_function_metric_invocations_daily.md:23]:
```suggestion
```

File: docs/tables/aws_lambda_function_metric_invocations_daily.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  timestamp,
+  sum
+from
+  aws_lambda_function_metric_invocations_daily
+order by
+  name,
+  timestamp;
+```
+
+
```
Comment (bigdatasourav) [inline docs/tables/aws_lambda_function_metric_invocations_daily.md:22]:
```suggestion
```

File: docs/tables/aws_lambda_function_metric_invocations_daily.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+```sql
+select
+  name,
+  timestamp,
+  round(sum::numeric,2) as sum_invocations,
+  sample_count
+from
+  aws_lambda_function_metric_invocations_daily
+where 
+    date_part('day', now() - timestamp) <=3
```
Comment (bigdatasourav) [inline docs/tables/aws_lambda_function_metric_invocations_daily.md:35]:
```suggestion
  date_part('day', now() - timestamp) <=3
```

**Review by bigdatasourav — CHANGES_REQUESTED** [review]
@sankeyraut Thank you for your interest and contribution. I have suggested few changes could you please check, also please provide some example query with output in the example query section.

**Comment by sankeyraut** [comment]
> @sankeyraut Thank you for your contribution. I have suggested few changes could you please check, also please provide some example query with output in the example query section.

Added sample query and output

**Review by bigdatasourav — APPROVED** [review]
LGTM

**Review by rajlearner17 — APPROVED** [review]
LGTM

**Comment by bigdatasourav** [comment]
@sankeyraut Thanks again for your contribution, the lambda function metric tables will be released this week.

**Pull request merged**